portal/views/portal.py

```diff
@@ -493,16 +484,28 @@ def website_consent_script(patient_id):
     redirect_url = request.args.get('redirect_url', None)
     user = current_user()
     org = user.first_top_organization()
-    if org:
-        terms = VersionedResource(app_text(
-            WebsiteConsentByOrg_ATMA.name_key(organization=org)))
-    else:
-        terms = VersionedResource(app_text(InitialConsent_ATMA.name_key()))
+    terms = get_terms(org)
     return render_template(
         'website_consent_script.html', user=user, terms=terms,
         entry_method=entry_method, redirect_url=redirect_url,
         patient_id=patient_id)
 
+
+def get_terms(org, role):
```

Comment:
It looks like you want to be able to call `get_terms()` with `org` and `role` as optional parameters.  If so, you need to give them default values, or line 487 is going to raise `TypeError: get_terms() takes exactly 2 arguments (1 given)`

simply change the definition to include the defaults:
`def get_terms(org=None, role=None)`